airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -49,6 +49,12 @@
   dockerImageTag: 0.3.4
   documentationUrl: https://hub.docker.com/r/airbyte/source-postgres
   icon: postgresql.svg
+- sourceDefinitionId: 9fa5862c-da7c-11eb-8d19-0242ac130003
+  name: Cockroachdb
+  dockerRepository: airbyte/source-postgres
+  dockerImageTag: 0.1.0
+  documentationUrl: https://hub.docker.com/r/airbyte/source-cockroachdb
+  icon: cockroachdb.svg
```

Comment:
there is currently a bug that causes airbyte to break when new icons are added. Let's remove the field for `icon` (but keep the icon itself in the repo) until this bug is fixed.

Reply:
Updated, thanks for letting me know